lib/spoom/sorbet.rb

```diff
@@ -89,21 +93,23 @@ def srb_version(*arg, path: ".", capture_err: false, sorbet_bin: nil)
         params(
           arg: String,
           path: String,
-          capture_err: T::Boolean,
           sorbet_bin: T.nilable(String),
         ).returns(T.nilable(T::Hash[String, Integer]))
       end
-      def srb_metrics(*arg, path: ".", capture_err: false, sorbet_bin: nil)
+      def srb_metrics(*arg, path: ".", sorbet_bin: nil)
         metrics_file = "metrics.tmp"
         metrics_path = "#{path}/#{metrics_file}"
-        T.unsafe(self).srb_tc(
+        result = T.unsafe(self).srb_tc(
           "--metrics-file",
           metrics_file,
           *arg,
           path: path,
-          capture_err: capture_err,
+          capture_err: true,
           sorbet_bin: sorbet_bin,
         )
+
+        raise Error::Metrics, result.err unless result.status
```

Comment:
Doesn't that mean we're going to raise for any type error? I think we want to raise only if the exit code == 137 (killed)? 

There is already a [constant to do something similar on segfaults](https://github.com/Shopify/spoom/blob/main/lib/spoom/sorbet.rb#L18). I think we could move all this logic to the [`srb` method](https://github.com/Shopify/spoom/blob/main/lib/spoom/sorbet.rb#L31) so it raises exceptions accordingly and thus clean the mess in https://github.com/Shopify/spoom/blob/main/lib/spoom/cli/bump.rb#L112-L118.